MIGRATION

```diff
@@ -0,0 +1,8 @@
+# Migration Notes
+
+API changes in MoveIt releases
```

Comment:
Also this file should *really* be documented somewhere everyone will look when transitioning their code.
Otherwise it might as well not exist. :warning: